src/sentry/runner/commands/init.py

```diff
@@ -12,9 +12,10 @@
 
 
 @click.command()
+@click.option('--dev/--no-dev', default=False, help='Use settings more conducive to local development.')
```

Comment:
I don't think we need the `--no-dev` variant here imo. Nobody will ever use that.